tests/comparetest.cpp

```diff
@@ -0,0 +1,75 @@
+#include <gtest/gtest.h>
+#include <fstream>
+#include <sched.h>
+#include <time.h>
+#include "cmdline.h"
+#include "gmgpolar.h"
+
+class Test_Parameters : public ::testing::Test
+{
+protected:
+    void SetUp() override
+    {
+        //Results of the run with default parameters. 
+        input.assign({5,0,0,4,4,3,0,1,0,3136,49,64,4,12}) ; 
```

Comment:
You don't have correct formatting. Please check for all files and commit potential adjustments.